frontend/.env

```diff
@@ -1 +1 @@
-VITE_API_URL=http://localhost
+VITE_API_URL=http://localhost:8000
```

Comment:
We might want to prevent file from being tracked by .git

Reply:
yeah. that was not supposed to go into this PR. thanks